client/src/components/ElementActions/PublishAction.js

```diff
@@ -122,7 +122,10 @@ const PublishAction = (MenuComponent) => (props) => {
     <MenuComponent {...props}>
       {props.children}
 
-      {(formDirty || !element.IsLiveVersion) && <AbstractAction {...newProps} />}
+      {props.element.canPublish === false
```

Comment:
You could do these sorts of checks without the `===` and ternary?
`{(formDirty || !element.IsLiveVersion) && props.element.canPublish && <AbstractAction {...newProps} />}`

Reply:
Wasn't super confident in the GraphQL stuff so hard checked `false` in case it canPublish didn't come through for some reason. But makes sense to assume it will always be there and shorthand this